python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -524,7 +536,10 @@ class MatMulLowBit(torch.autograd.Function):
     def forward(ctx, A, weight, input_seq_size):
         ctx.is_empty = False
         import linear_q4_0
-        result = linear_q4_0.forward_new(A, weight.data, weight.qtype, input_seq_size)
+        if hasattr(weight, "enable_deepspeed_zero3") and weight.enable_deepspeed_zero3:
+            result = linear_q4_0.forward_new(A, weight.data.byte(), NF4, input_seq_size)
```

Comment:
`weight.data.byte()` means converting every element to `torch.uint8`. Do you mean something like `weight.data.view(torch.uint8)`?

Reply:
And shouldn't the `qtype` be the same as the `load_in_low_bit` in `from_pretrained`?